compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategies.cpp

```diff
@@ -729,7 +856,8 @@ static LogicalResult buildReductionCpuStrategy(
 
   // Step 1: Distribute to blocks using the current IREE lowering config.
   variantH = buildReductionStrategyBlockDistributionPart(
-      b, variantH, originalFillH, originalGenericH, tileSizes0Generic);
+      b, variantH, originalFillH, originalGenericH, originalGenericH,
```

Comment:
the doubling of handles and action at a distance feels tricky to maintain, how about a potentially null `maybeXXX`  ?

Reply:
Done.